opentelemetry-api/TrackAdvancedException/src/main/java/com/example/TrackAdvancedException.java

```diff
@@ -0,0 +1,61 @@
+package com.example;
+
+import com.azure.monitor.opentelemetry.exporter.AzureMonitorExporterBuilder;
+import com.azure.monitor.opentelemetry.exporter.implementation.ResourceAttributes;
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.logs.GlobalLoggerProvider;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.logs.SdkLoggerProvider;
+import io.opentelemetry.sdk.logs.export.BatchLogRecordProcessor;
+import io.opentelemetry.sdk.logs.export.LogRecordExporter;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.samplers.Sampler;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class TrackAdvancedException {
+
+  private static final String CONNECTION_STRING = "<Your Connection String>";
+  private static final Logger log4jLogger = LogManager.getLogger("log4j-logger");
+
+  public static void main(String[] args) throws InterruptedException {
+    initOpenTelemetry();
+
+    trackAdvancedException();
+    Thread.sleep(6000); // wait at least 5 seconds to give batch span processor time to export
+  }
+
+  private static void trackAdvancedException() {
+    Exception exception = new Exception(("my exception"));
+    StackTraceElement[] stackTraceElements = new StackTraceElement[] {
+        new StackTraceElement("classLoaderName", "moduleName", "1.0", "declaringClass1", "methodName1", "filename1", 123),
+        new StackTraceElement("classLoaderName", "moduleName", "1.0", "declaringClass2", "methodName2", "filename2", 456),
+        new StackTraceElement("classLoaderName", "moduleName", "1.0", "declaringClass3", "methodName3", "filename3", 789)
```

Comment:
this is more common:

```suggestion
        new StackTraceElement("declaringClass3", "methodName3", "filename3", 789)
```

Reply:
apply to all?

Reply:
ya